### DISCUSSION TITLE
How can I flip an image upside down?

### DISCUSSION THREAD

**Discussion by soopermodel** [SixLabors/ImageSharp#2302] [Q&A]
```
string pictoPath = "/root/test.png";
byte[] bgraData;
var image = Image.Load<Bgra32>(pictoPath);
var memGroup = image.GetPixelMemoryGroup().ToArray()[0];
bgraData = MemoryMarshal.AsBytes(memGroup.Span).ToArray();
```
hi, i'm importing a PNG in BGRA32 format and convert it to a bytearray to use on another library. i wonder how i can flip the image upside down. i tried `RotateExtensions.Rotate(image, RotateMode.Rotate90)` but i get this
```
Argument 1: cannot convert from 'SixLabors.ImageSharp.Image<SixLabors.ImageSharp.PixelFormats.Bgra32>' to 'SixLabors.ImageSharp.Processing.IImageProcessingContext'
```

**Comment by JimBobSquarePants** [comment]
You’re trying to pass the wrong type to an extension method. There’s some examples in the docs here which should help you see how we process images. 

https://docs.sixlabors.com/articles/imagesharp/processing.html
